proto/axelar/axelarnet/v1beta1/events.proto

```diff
@@ -74,15 +74,47 @@ message FeeCollected {
 }
 
 message ContractCallSubmitted {
-  string sender = 1;
-  string source_chain = 2
+  string message_id = 1 [ (gogoproto.customname) = "MessageID" ];
+  string sender = 2;
```

Comment:
i don't like that the irrelevant metadata (foreign assets supported, key type etc.) of each chain is added to the event. I agree it would simplify the struct, but i think it reduces the readability of the event too. also makes it harder to change later due to consumers of this event, and we'd have to keep around a deprecated blob.

Reply:
fair points, I will change it back. should change to `CrossChainAddress`  once we refactor `Chain` to `ChainName` in `CrossChainAddress `